tasks/login.yml

```diff
@@ -1,7 +1,4 @@
 ---
 - name: Login to docker hub
-  docker_login:
-    username: "{{ docker_pi_hub_username }}"
-    password: "{{ docker_pi_hub_password }}"
-    email: "{{ docker_pi_hub_email }}"
+  command: docker login --username="{{ docker_pi_hub_username }}" --password="{{ docker_pi_hub_password }}"
```

Comment:
Since email isn't needed anymore could you also remove it from the main task file and default variables? And update the README?